ElementX/Sources/Screens/Authentication/ServerSelection/MockServerSelectionScreenState.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+enum MockServerSelectionScreenState: CaseIterable {
+    case matrix
+    case emptyAddress
+    case invalidAddress
+    case nonModal
+    
+    /// Generate the view struct for the screen state.
+    var viewModel: ServerSelectionViewModel {
+        // swiftlint:disable:next implicit_getter
+        @MainActor get {
```

Comment:
```suggestion
    @MainActor var viewModel: ServerSelectionViewModel {
           ...
```
It seems there is no obstacle to make the property `@MainActor` and get rid of the getter.

Reply:
Questioning why I even thought to reach for this style now 🤔